example/docker-compose/grafana/dashboards/agent-operational.json

```diff
@@ -284,7 +284,7 @@
                "thresholds": [ ],
                "timeFrom": null,
                "timeShift": null,
-               "title": "CPU",
+               "title": "CPU Usage [time/s]",
```

Comment:
The only problem with including units in the title that I can think of, is that if someone edits an axis on the dashboard to have different units it might be confusing, or they'd have to update the title manually. 

Wouldn't it be better to just set the unit in the Grafana settings for each axis, and then Grafana will just display them on each axis?

Reply:
I think if someone changes the axis, they can also change the title. I didn't put unit everywhere as it's sometimes obvious (e.g. bytes), but I think it's helpful for some less obvious charts, like CPU Usage which is "how many seconds of CPU time was used every second".

Reply:
Even in those cases I think I'd prefer to have the units on the actual axes instead of in the title. But I'm ok with having this in the title like this, it's still an improvement and I think it'll work ok.